ubuntu_image/image.py

```diff
@@ -55,17 +55,15 @@ def copy_blob(self, blob_path, **dd_args):
         :type blob_path: str
         """
         # Put together the dd command.
-        args = ['dd', 'of={}'.format(self.path), 'if={}'.format(blob_path),
-                'conv=sparse']
-        for key, value in dd_args.items():
-            args.append('{}={}'.format(key, value))
+        args = SPACE.join('{}={}'.format(key, value)
+                          for key, value in dd_args.items())
         # Run the command.  We'll capture stderr for logging purposes.
         #
         # TBD:
         # - check status of the returned CompletedProcess
         # - handle errors
         # - log stdout/stderr
-        run(args, stdout=PIPE, stderr=PIPE, universal_newlines=True)
+        run('dd of={} if={} {} conv=sparse'.format(self.path, blob_path, args))
```

Comment:
No, this is a regression in support for path names that contain spaces.  We should always prefer arg arrays over a space-split string where possible, especially where we are dealing with arguments including paths that the user can influence.

Will fix up on the branch and push for re-testing.